### ISSUE TITLE
PureComponents are allowed

### ISSUE THREAD

**Issue by eBitzu** [tatethurston/eslint-plugin-react-prefer-function-component#9]
Hi, this is not flagged as wrong:

export class SomeRandomStuff extends PureComponent {}

Can I PR a change here?

PS: I can't push branches on this repo

**Comment by tatethurston** [comment]
Hi @eBitzu thanks for reporting this. Happy to accept a PR.

Yes, generally open source repositories don’t accept branches from everyone. You can fork the repo and push a branch to your fork, and then open the PR from your branch.

**Comment by tatethurston** [comment]
Hi @eBitzu I'm unable to recreate this issue on version `3.0.0`. What version of this library are you using?

The following is flagged:
```jsx
  import { PureComponent } from 'react';
  export class SomeRandomStuff extends PureComponent {}
```
